float64 distance
float64 angle
---
memoria/ErrorMsg error